lemur/certificates/cli.py

```diff
@@ -798,6 +798,8 @@ def disable_rotation_of_duplicate_certificates(commit):
         return
 
     log_data["authorities"] = authority_names
+    days_since_issuance = current_app.config.get("DAYS_SINCE_ISSUANCE_DISABLE_ROTATE_OF_DUPLICATE_CERTIFICATES", -1)
```

Comment:
would `none` serve better in this case, or maybe a reasonable defaults like `90` days?

Reply:
I'll change to `none` which will take care of all duplicate certificates. 90 Days is a good option too given LE certs are valid for 90 days. But `none` is more generic.